NOTICE

```diff
@@ -0,0 +1,5 @@
+Apache TinkerPop
+Copyright 2015-2021 The Apache Software Foundation.
+
+This product includes software developed at
+The Apache Software Foundation (http://www.apache.org/).
```

Comment:
I'm not sure if we need a notice ? can't find any mention on the docs, maybe you can help me pointing at it ?